apps/admin-x-activitypub/src/components/feed/ArticleModal.tsx

```
@@ -864,23 +868,29 @@ const ArticleModal: React.FC<ArticleModalProps> = ({
-                                            type='Note'
-                                            onClick={() => {
-                                                navigateForward(item.id, item.object, item.actor, false);
-                                            }}
-                                            onCommentClick={() => {
-                                                navigateForward(item.id, item.object, item.actor, true);
-                                                setIsFocused(true);
-                                            }}
-                                        />
+                                        {
+                                            isEnabled('deleteButton') && item.object.type === 'Tombstone' ? (
+                                                <DeletedFeedItem last={false} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)